posts/2024-02-08-interviews-anirban_chetia/index.qmd

```diff
@@ -29,7 +29,7 @@ Forms of bringing in people are mostly subjected to one’s interests, which com
 
 - Making one a better programmer. This comes to be not just by contributing directly, but also by learning from others’ contributions.
 
-- Financial gain.
+- Financial gain or incentives from working on the project.
```

Comment:
Can you give an example? do they think they will get a better paying job? or?

Reply:
How about Matt being paid by H20.ai previously?

> do they think they will get a better paying job? or?

Yup, people think that they can get better job offers by being proficient at `data.table` primarily due to it saving time in doing analyses for data science tasks.

Reply:
Right, I did not (so it's just something we know of in general, but not gathered from the interviews - should I still mention that?)